templates/f5/1/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+external-lb:
+  image: rancher/external-lb:v0.3.6
+  command: -provider=f5_BigIP
+  expose: 
+   - 1000
+  environment:
+    F5_BIGIP_HOST: ${F5_BIGIP_HOST}
+    F5_BIGIP_USER: ${F5_BIGIP_USER}
+    F5_BIGIP_PWD: ${F5_BIGIP_PWD}
+    LB_TARGET_RANCHER_SUFFIX: ${LB_TARGET_RANCHER_SUFFIX}
```

Comment:
Getting error message `level=fatal msg="Failed to initialize Rancher API client: Environment variable 'CATTLE_URL' is not set"`

Could you please add label to get `CATTLE_URL` initialized??

```
+  labels:
+    io.rancher.container.create_agent: "true"
+    io.rancher.container.agent.role: "external-dns"
```